File: Level 1/바탕화면 정리/solution.py
def solution(wallpaper):
    x_arr = []
    y_arr = []

    for y in range(len(wallpaper)):
        for x in range(len(wallpaper[0])):
            if wallpaper[y][x] == "#":
                x_arr.append(x)
                y_arr.append(y)

    return [min(y_arr), min(x_arr), max(y_arr) + 1, max(x_arr) + 1]

File: Level 1/바탕화면 정리/solution.js
function solution(wallpaper) {
  const xArr = [];
  const yArr = [];

  wallpaper.forEach((row, y) => {
    [...row].forEach((_, x) => {
      if (wallpaper[y][x] === "#") {
        xArr.push(x);
        yArr.push(y);
      }
    });
  });

  return [
    Math.min(...yArr),
    Math.min(...xArr),
    Math.max(...yArr) + 1,
    Math.max(...xArr) + 1,
  ];
}
